src/components/Slogan/Slogan.tsx

```diff
@@ -2,17 +2,8 @@ import * as React from 'react';
 
 import './Slogan.scss';
 
-export class Slogan extends React.Component {
-  readonly  state = {
-    slogan: 'Stay connected.'
-  };
-
-  public render() {
-    return (
-      <div className="slogan">
-        <h1>{this.state.slogan}</h1>
-      </div>
-    );
-  }
-}
-
+export const Slogan = () => (
+  <div className="slogan">
+    <h1>Stay connected.</h1>
+  </div>
+);
```

Comment:
I'm not sure if we are using this slogan elsewhere. If no, we don't need a separate component for it.